src/chainstate/burn/db/sortdb.rs

```diff
@@ -858,6 +861,19 @@ const SORTITION_DB_SCHEMA_5: &'static [&'static str] = &[
     );"#,
 ];
 
+// force the node to go and store the memo for stack_stx and delegate_stx
+const SORTITION_DB_SCHEMA_6: &'static [&'static str] = &[
+    r#"
+    ALTER TABLE stack_stx ADD COLUMN memo TEXT;
+    "#,
+    r#"
+    ALTER TABLE delegate_stx ADD COLUMN memo TEXT;
+    "#,
```

Comment:
Per my comment above -- this alter table syntax will fill the table with `NULL` sqlite values in this column. This can be avoided with defaults:

```sql
ALTER TABLE stack_stx ADD COLUMN memo TEXT NOT NULL DEFAULT "";
```

```sql
ALTER TABLE delegate_stx ADD COLUMN memo TEXT NOT NULL DEFAULT "";
```

This would still need a test case to ensure that the pre-schema-6 values are still parseable from the DB (they should just have a zero-length memo field).

Reply:
Hmm.. Personally I'm in favor of dropping the table and re-adding it, instead of doing `ALTER TABLE`.  

Otherwise, in `insert_stack_stx`, I would need to case the db query on the version of the Sortition database (which feels clunky to me).